[Bug][DataProxy] Build failed

### What happened

When I use `mvn clean install -DskipTests` to build the project, it failed.
```java
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:2.5.3:single (assemble) on project dataproxy-dist: 
Execution assemble of goal org.apache.maven.plugins:maven-assembly-plugin:2.5.3:single failed: 
group id '110520945' is too big ( > 2097151 ) -> [Help 1]
```

It seems it's better to use posix rather than gnu, since gnu may cause some issues on some platforms.
https://issues.apache.org/jira/browse/MPOM-132


### What you expected to happen

Build success

### How to reproduce

mvn clean install -DskipTests

### Environment

Maven version: 3.8.3
Java version: 1.8
OS: maxOs Catalina 10.15.7

### InLong version

master

### InLong Component

InLong DataProxy

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/inlong-agent/agent-release/pom.xml
+++ b/inlong-agent/agent-release/pom.xml
@@ -65,7 +65,7 @@
                             <descriptors>
                                 <descriptor>assembly.xml</descriptor>
                             </descriptors>
-                            <tarLongFileMode>gnu</tarLongFileMode>
+                            <tarLongFileMode>posix</tarLongFileMode>
                         </configuration>
                     </execution>
                 </executions>

--- a/inlong-dataproxy/dataproxy-dist/pom.xml
+++ b/inlong-dataproxy/dataproxy-dist/pom.xml
@@ -50,7 +50,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-assembly-plugin</artifactId>
-                <version>2.5.3</version>
+                <version>${plugin.assembly.version}</version>
                 <executions>
                     <execution>
                         <id>assemble</id>
@@ -63,7 +63,7 @@
                             <descriptors>
                                 <descriptor>src/main/assembly/assembly.xml</descriptor>
                             </descriptors>
-                            <tarLongFileMode>gnu</tarLongFileMode>
+                            <tarLongFileMode>posix</tarLongFileMode>
                         </configuration>
                     </execution>
                 </executions>

--- a/inlong-dataproxy/pom.xml
+++ b/inlong-dataproxy/pom.xml
@@ -41,6 +41,7 @@
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
         <flume.version>1.9.0</flume.version>
+        <plugin.assembly.version>3.2.0</plugin.assembly.version>
         <netty.version>3.8.0.Final</netty.version>
         <codec.version>1.15</codec.version>
         <servlet.version>2.5-20110124</servlet.version>

--- a/inlong-tubemq/pom.xml
+++ b/inlong-tubemq/pom.xml
@@ -76,6 +76,7 @@
         <mockito.version>2.18.0</mockito.version>
         <powermock.version>2.0.4</powermock.version>
         <exec.maven.version>1.6.0</exec.maven.version>
+        <plugin.assembly.version>3.2.0</plugin.assembly.version>
         <build.helper.maven.version>3.0.0</build.helper.maven.version>
         <protobuf.version>2.5.0</protobuf.version>
         <je.version>7.3.7</je.version>

--- a/inlong-tubemq/tubemq-client/pom.xml
+++ b/inlong-tubemq/tubemq-client/pom.xml
@@ -35,13 +35,13 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-assembly-plugin</artifactId>
-                <version>2.4</version>
+                <version>${plugin.assembly.version}</version>
                 <configuration>
                     <finalName>apache-inlong-tubemq-client-${project.version}</finalName>
                     <descriptors>
                         <descriptor>src/main/assembly/assembly.xml</descriptor>
                     </descriptors>
-                    <tarLongFileMode>gnu</tarLongFileMode>
+                    <tarLongFileMode>posix</tarLongFileMode>
                 </configuration>
                 <executions>
                     <execution>

--- a/inlong-tubemq/tubemq-example/pom.xml
+++ b/inlong-tubemq/tubemq-example/pom.xml
@@ -35,13 +35,13 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-assembly-plugin</artifactId>
-                <version>2.4</version>
+                <version>${plugin.assembly.version}</version>
                 <configuration>
                     <finalName>apache-inlong-tubemq-example-${project.version}</finalName>
                     <descriptors>
                         <descriptor>src/main/assembly/assembly.xml</descriptor>
                     </descriptors>
-                    <tarLongFileMode>gnu</tarLongFileMode>
+                    <tarLongFileMode>posix</tarLongFileMode>
                 </configuration>
                 <executions>
                     <execution>

--- a/inlong-tubemq/tubemq-server/pom.xml
+++ b/inlong-tubemq/tubemq-server/pom.xml
@@ -53,13 +53,13 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-assembly-plugin</artifactId>
-                <version>2.4</version>
+                <version>${plugin.assembly.version}</version>
                 <configuration>
                     <finalName>apache-inlong-tubemq-server-${project.version}</finalName>
                     <descriptors>
                         <descriptor>src/main/assembly/assembly.xml</descriptor>
                     </descriptors>
-                    <tarLongFileMode>gnu</tarLongFileMode>
+                    <tarLongFileMode>posix</tarLongFileMode>
                 </configuration>
                 <executions>
                     <execution>

